src/main/resources/entity-types/inventory/simple_instance_type.json5

```diff
@@ -0,0 +1,138 @@
+{
+  id: 'af44e2e0-12e0-4eec-b80d-49feb33a866c',
+  name: 'simple_instance_type',
+  private: true,
+  sources: [
+    {
+      type: 'db',
+      alias: 'instance_type',
+      target: 'src_inventory_instance_type',
+    },
+  ],
+  requiredPermissions: [
+    'inventory-storage.instance-types.collection.get',
+    'inventory-storage.instance-types.item.get'
+  ],
+  columns: [
+    {
+      name: 'id',
```

Comment:
I opted not to bother adding a `valueSourceApi` for `id` even though we could. I figure that users wouldn't really benefit from a list of UUIDs (especially when they can use `code` or `name` to search easily). I assume that if someone were to search by ID, they would probably be copying/pasting from somewhere. And in that case, there's a solid chance that they are searching for an ID that doesn't exist.